internal/github/checkPendingCI.go

```diff
@@ -0,0 +1,127 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/google/go-github/v60/github" // Ensure your go-github library version matches
+
+	"github.com/chia-network/github-bot/internal/config"
+)
+
+// CheckForPendingCI returns a list of PR URLs that are ready for CI to run but haven't started yet.
+func CheckForPendingCI(githubClient *github.Client, internalTeam string, cfg config.CheckStalePending) ([]string, error) {
+	teamMembers, _ := GetTeamMemberList(githubClient, internalTeam)
+	var pendingPRs []string
+
+	for _, fullRepo := range cfg.CheckStalePending {
+		log.Println("Checking repository:", fullRepo.Name)
+		parts := strings.Split(fullRepo.Name, "/")
+		if len(parts) != 2 {
+			return nil, fmt.Errorf("invalid repository name - must contain owner and repository: %s", fullRepo.Name)
```

Comment:
If someone gives an invalid repo name to the config, do we want it to completely stop here, like it currently is? Or can we continue on to the next repo in the config?

Reply:
I added a continue and changed to printing out the error